src/apps/interlink/receiver.lua

```diff
@@ -0,0 +1,57 @@
+-- Use of this source code is governed by the Apache 2.0 license; see COPYING.
+
+module(...,package.seeall)
+
+local shm = require("core.shm")
+local interlink = require("lib.interlink")
+
+local Receiver = {name="apps.interlink.Receiver"}
+
+function Receiver:new ()
```

Comment:
I think the interlink apps would do better to use the standard way to declare app parameters.  It could default to the app name for the link name, but the interlink name should be configurable.  WDYT?

Reply:
Agree, have a patch for that in the barrel already.

Reply:
Agree, have a patch just for that already in the barrel.

About the other bits (to which I can for some reason not reply individually)...

- It’s not like `max_packets` was configurable at run time before, so I see it as a net-zero to change it to your formulation, which I might also prefer. Replacing these `uint64_t`s with 32 bit ints seems like a good idea, regardless. Both of these don’t appear to me to be related to this PR in particularily, so I tend to pass.

- Yeah I could shuffle around the code some... but I honestly don’t see the benefit right now.

Let’s get this merged and improve on it incrementally?